quickvision/models/detection/faster_rcnn/engine.py

```diff
@@ -242,7 +242,29 @@ def train_sanity_fit(model, train_loader,
         log_interval : (optional) Defualt 100. Integer to Log after specified batch ids in every batch.
         fp16: : (optional) If True uses PyTorch native mixed precision Training.
     """
-    pass
+    
+    model = model.to(device)
+    model.train()
+
+    cnt = 0
+
+    for batch_idx, (input, targets) in enumerate(train_loader):
+        
+        images = list(image.to(device) for image in outputs)
+        targets = [{k: v.to(device) for k, v in t.items()} for t in targets]
+
+        loss_dict = model(images, targets)
+
+        cnt += 1
+
+        if num_batches is not None:
+            if cnt >= num_batches:
+                print(f"Done till {num_batches} train batches")
```

Comment:
```
print("All specicied batches done.")
val_sanity_end = time.time()
print(f"Validation Sanity check passed in time {val_sanity_end - val_sanity_start} !!")
```

Reply:
But function description mentions time and metrics are not tracked. 
I will add the time calculation too

Reply:
Time is fine, it just shows a rough estimate of code. Sometimes it is too long to do a sanity fit too.
The `log_interval` would then keep showing status to user.